engine/src/em-stack.cpp

```diff
@@ -114,29 +116,79 @@ bool MCStack::view_platform_dirtyviewonresize() const
 	return true;
 }
 
-void
-MCStack::view_platform_updatewindow(MCRegionRef p_dirty_region)
+
+static MCStackUpdateCallback s_updatewindow_callback = nullptr;
```

Comment:
I suggest moving these statics to just after the comment above as they are a part of the 'view management' section.

Reply:
Moved :)